packages/google_sign_in/google_sign_in_platform_interface/lib/src/types.dart

```diff
@@ -22,6 +22,42 @@ enum SignInOption {
   games
 }
 
+/// Plugin parameters that are provided at initialization.
+class GoogleSignInParams {
+  /// Uses the given data to construct an instance.
+  GoogleSignInParams({
+    this.scopes = const <String>[],
+    this.signInOption = SignInOption.standard,
+    this.hostedDomain,
+    this.clientId,
+    this.serverClientId,
+  });
+
+  /// The list of OAuth scope codes to request when signing in.
+  ///
+  /// These scope codes will determine the level of data access that is granted
+  /// to your application by the user. The full list of available scopes can be
+  /// found here: <https://developers.google.com/identity/protocols/googlescopes>
+  final List<String> scopes;
+
+  /// Default configuration option to use when signing in.
+  ///
+  /// [SigninOption.games] is only supported on Android.
+  final SignInOption signInOption;
+
+  /// The hosted domain to which user account are restricted.
+  ///
+  /// By setting this, sign in will be restricted to accounts of the user in the
+  /// specified domain. By default, the list of accounts will not be restricted.
+  final String? hostedDomain;
+
+  /// The OAuth client ID of the app.
```

Comment:
Let's also mention what does a null value mean and the default is null.

Reply:
Done